.github/workflows/auto-pr.yml

```diff
@@ -0,0 +1,27 @@
+name: Auto PR for new versions
+
+on:
+  schedule:
+  - cron: 0 0 * * *
+
+jobs:
+  update:
+    strategy:
+      matrix:
+        version:
+        - 10
+        - 12
+        - 14
```

Comment:
this will create 3 PRs if there are updates for all release lines, right? Should we instead just do `./update.sh` without passing a version and use some node script to construct the PR title and body?

Reply:
`./update.sh` is broken atm and just runs against all versions even when you pass one as an argument.
Also, this will create PRs for yarn updates too even when there isn't a node update, since it doesn't have the `-s` flag. 

I am interested in fixing update.sh or even perhaps porting it to a node script, if that is desired. It could be made more useful like returning what node versions it updated, etc.

Reply:
migrating to a js script makes sense to me.

As a temporary workaround, the script could run with `-s` first, and only if there is a diff, run again without the `-s`. But yeah, might make more sense to port it to a js script so it's a bit easier to work with more advanced logic?

Reply:
What should happen if there is a keys update (like there was a few days ago). Should any images be updated if there are no node updates?

Reply:
Good point, it should only create a PR if the Node version line changes. Key changes should only go in with a version bump

Reply:
Alright, I'll add some logic to grab the keys in the existing files to replay them on the templates, unless the node version changes.

Reply:
Can probably just try looking at the `git diff` result

Reply:
> it should only create a PR if the Node version line changes

There might be bug fixes and such, so I think it's better to say that

1. yarn version change
1. key changes

only triggers an update _if_ the node version changes - all other changes than those two should always trigger